readthedocs/projects/tasks.py

```diff
@@ -595,6 +602,15 @@ def set_valid_clone(self):
         self.project.has_valid_clone = True
         self.version.project.has_valid_clone = True
 
+    def save_build_config(self):
+        """Save config in the build object."""
+        pk = self.build['id']
+        build_data = api_v2.build(pk).get()
+        config = self.config.as_dict()
+        build_data['config'] = config
+        api_v2.build(pk).put(build_data)
+        self.build['config'] = config
```

Comment:
I don't think we are going to use this here, but just in case, it also helps when testing this, write a test for this was hard :disappointed:

Reply:
Yea, testing just about everything about our builds is tricky :/